libs/hscim/src/Web/Scim/Schema/User.hs

```diff
@@ -329,7 +331,7 @@ applyUserOperation user (Operation Replace Nothing (Just value)) = do
             externalId = externalId u,
             active = active u
           }
-    _ -> throwError (badRequest InvalidPath (Just "we only support attributes username, displayname, externalid, active"))
+    _ -> throwError (badRequest InvalidPath (Just "we only support attributes username, displayname, externalid, active, roles"))
```

Comment:
i'm itching to give the supported-fields list in line 325 above a name, and construct this and other suitable error messages from that.  but neither important nor urgent i guess, so fuck it?  :-)